gwpy/cli/timeseries.py

```diff
@@ -55,7 +56,9 @@ def get_suptitle(self):
 
     def get_title(self):
         suffix = super(TimeSeries, self).get_title()
-        rates = {ts.sample_rate for ts in self.timeseries}
+        # limit significant digits for minute trends
+        rates = {round(ts.sample_rate.value, 3) * units.Hz
+                 for ts in self.timeseries}
```

Comment:
I'm sorry @areeda, it turns out that Quantities have built-in support for rounding, so can you try using this:

```python
        rates = {ts.sample_rate.round(3) for ts in self.timeseries}
```

Reply:
How's this/